tket/include/tket/Utils/Expression.hpp

```diff
@@ -19,14 +19,28 @@
  * @brief Functions related to (possibly symbolic) phase values
  */
 
+#include <symengine/eval.h>
+#include <symengine/expression.h>
+#include <symengine/functions.h>
+#include <symengine/integer.h>
+#include <symengine/visitor.h>
+
 #include <map>
 #include <optional>
 #include <set>
 #include <vector>
 
 #include "Constants.hpp"
 #include "Json.hpp"
-#include "Symbols.hpp"
+
```

Comment:
These lines are duplicated below (not on `main`). Perhaps a merge resolution issue?

Reply:
I tried to do an automatic merge with solving all conflicts with what is on main. I fixed this, and I carefully checked everything else, hope this is better now :pray: